src/MessagingInput/MessagingInput.tsx

```diff
@@ -77,7 +77,7 @@ const MessagingInputRenderFunction: React.ForwardRefRenderFunction<MessagingInpu
           <>
             <img
               className={cssClass("SendIcon")}
-              alt="messaging settings"
+              alt="Send message"
```

Comment:
@arsalansufi Not to beat a dead horse, but alt text is another place where we have text in clever-components that would need to be translated.